packages/office-ui-fabric-react/src/components/Modal/Modal.tsx

```diff
@@ -121,6 +122,7 @@ export class Modal extends BaseComponent<IModalProps, IDialogState> {
             <div className={ modalClassName }>
               <Overlay isDarkThemed={ isDarkOverlay } onClick={ isBlocking ? undefined : (onDismiss as any) } />
               <FocusTrapZone
+                componentRef={ ref => this._focusTrapZone = ref }
```

Comment:
instead of lambda, use `this._resoolveRef('_focusTrapZone')`

Reply:
Aha, thanks!